.firebase.json

```diff
@@ -1,7 +1,12 @@
 {
   "emulators": {
     "auth": {
+      "host": "0.0.0.0",
```

Comment:
As per our discussion, check whether `0.0.0.0` is needed or `localhost` should work? (Logically we are expecting localhost to work!)